project/src/main/career/career-level-library.gd

```diff
@@ -24,6 +24,10 @@ func all_level_ids() -> Array:
 	for region in regions:
 		for career_level in region.levels:
 			result[career_level.level_id] = true
+		if region.boss_level:
+			result[region.boss_level.level_id] = true
+		if region.intro_level:
+			result[region.intro_level.level_id] = true
```

Comment:
Document this change.

Reply:
Added commit comment, `CareerLevelLibrary.all_level_ids now includes boss, intro level`